src/Radio/RadioGroup.d.ts

```diff
@@ -8,7 +8,7 @@ export interface RadioGroupProps extends StandardProps<
   'onChange'
 > {
   name?: string;
-  onChange?: (event: React.ChangeEvent<{}>, value: string) => void;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>, value: string) => void;
```

Comment:
Is this one accurate? Based on how it's used,

```js
<FormGroup data-mui-test="RadioGroup" role="radiogroup" {...other}>
  {React.Children.map(children, (child, index) => {
    if (!React.isValidElement(child)) {
      return null;
    }

    return React.cloneElement(child, {
      key: index,
      name,
      inputRef: node => {
        if (node) {
          this.radios.push(node);
        }
      },
      checked: value === child.props.value,
      onChange: this.handleRadioChange,
    });
  })}
</FormGroup>
```

it seems like it can be used with arbitrary children.